Packs/AWS_Bedrock/Integrations/AwsBedrock/README.md

```diff
@@ -0,0 +1,5 @@
+This README contains the full documentation for your integration.
+
+You auto-generate this README file from your integration YML file using the `demisto-sdk generate-docs` command.
```

Comment:
Please generate a README.md file for your integration/script.
To do that, please run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.